app/controllers/bookmarks_controller.rb

```diff
@@ -19,6 +19,7 @@ def create
     @bookmark = Bookmark.new(bookmark_params)
 
     if @bookmark.save
+      flash[:success] = "Bookmark successfully created!"
       redirect_to @bookmark
     else
```

Comment:
@bion I'm having trouble getting the error messages into their own flash.  It seems like I should be able to put something here in the `else` branch (maybe a private method so I can share it with `update()`) like
 `flash.now[:danger] = @bookmark.errors.full_messages` 
and then use a view to structure those messages.  

It looks like flashes are usually used for a single message, 
i.e. `flash[:danger] = "Invalid title/link"`
 but there are several potential errors provided by the validations and I want to be able to list them in the flash.